community-operators/metering/metering-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -6,14 +6,15 @@ metadata:
   name: metering-operator.v0.13.0
   annotations:
     tectonic-visibility: openshift-feature
+    categories: "OpenShift Optional, Monitoring"
   namespace: placeholder
   labels:
     alm-catalog: openshift-feature
     operator-metering: "true"
 spec:
   displayName: Metering
   description: Metering can generate reports based on historical usage data from a cluster, providing accountability for how resources have been used.
-  keywords: [metering metrics reporting]
+  keywords: ['metering', 'metrics', 'reporting', 'prometheus', 'chargeback']
```

Comment:
I'm not sure if there's an agreement whether this should be a list of strings or a comma-separated string.

Reply:
It's defined as an array of string - https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go#L140.

Reply:
I believe I moved all of then to a list of strings, so at least they are consistent